cli/jobs/pipelines/automl/cli-automl-classification-task-bankmarketing-pipeline/classification-task-bankmarketing-pipeline.yml

```diff
@@ -43,3 +42,11 @@ jobs:
       mltable_output: ${{parent.jobs.classification_node.outputs.best_model}}
     command: ls ${{inputs.mltable_output}}
     environment: azureml:AzureML-sklearn-0.24-ubuntu18.04-py37-cpu:1
+  register_model_job:
```

Comment:
(nit) rename to `register_model_node`

Reply:
Also, we can rem ove the `show_output` node if we haven't already.

Reply:
Changes done